.github/workflows/ci.yml

```diff
@@ -45,7 +45,7 @@ jobs:
           make -C target/snitch_cluster sw
       - name: Build Hardware
         run: |
-          make -C target/snitch_cluster bin/snitch_cluster.vlt
+          make -C target/snitch_cluster bin/snitch_cluster.vlt -j$(nproc)
```

Comment:
make will spawn an unlimited amount of jobs if I don't put `nproc` there. This might overload the system. We can experiment with this in a next PR maybe?